utils/windowfunction/windowfunctiontype.cpp

```diff
@@ -337,6 +357,11 @@ template<> void WindowFunctionType::setValue<long double>(uint64_t i, long doubl
     fRow.setLongDoubleField(t, i);
 }
 
+template<> void WindowFunctionType::setValue<int128_t>(uint64_t i, int128_t& t)
+{
+    fRow.setInt128Field(t, i);
+}
```

Comment:
It probably should be const, but since the parent template isn't const ref, this can't be unless we go back and change a bunch. It's probably not worth the effort. const only exists to protect programmers from programmers. The machine don't care.

Reply:
Added uint128_t stuff. Required to add to Row as well.